### PR TITLE
Add tolerance percentage to Piastrix

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

File: openapi/components/schemas/GatewayAccountConfig/Piastrix.yaml [hunk 1]
```diff
@@ -19,5 +19,15 @@ allOf:
...
           - secretKey
+      settings:
+        type: object
+        description: Piastrix settings object.
+        properties:
+          tolerancePercentage:
+            type: integer
+            description: Tolerance percentage for settled amount.
+            minimum: 0
+            maximum: 5
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/Piastrix.yaml:30]: I wonder why it is different compared to Paymero https://github.com/Rebilly/api-definitions/pull/332/

Comment (dara123) [reply openapi/components/schemas/GatewayAccountConfig/Piastrix.yaml:30]: we fixed Paymero. should be max 5

Comment (dara123) [reply openapi/components/schemas/GatewayAccountConfig/Piastrix.yaml:30]: https://github.com/Rebilly/api-definitions/blob/master/openapi/components/schemas/GatewayAccountConfig/Paymero.yaml#L28

**Review by ashkarpetin — APPROVED** [review]

**Review by slavcodev — COMMENTED** [review]
Please re-use definition https://github.com/Rebilly/api-definitions/blob/6db1f07d41faea89059edd5e1b5825856b8fdb3a/openapi/components/schemas/GatewayAccountConfig/UPayCard.yaml#L33

**Comment by dara123** [comment]
I don't think that work yet. I still see bug issue for this open

**Review by adamaltman — APPROVED** [review]

File: openapi/components/schemas/GatewayAccountConfig/Piastrix.yaml [hunk 2]
```diff
@@ -19,5 +19,15 @@ allOf:
...
+            example: 5
```
Comment (adamaltman) [inline openapi/components/schemas/GatewayAccountConfig/Piastrix.yaml:31]:
```suggestion
           $ref: ../Gateways/AmountAdjustmentTolerance.yaml
```

**Review by adamaltman — CHANGES_REQUESTED** [review]

File: openapi/components/schemas/GatewayAccountConfig/Piastrix.yaml (hunk 2 above)
Comment (dara123) [reply openapi/components/schemas/GatewayAccountConfig/Piastrix.yaml:31]: Using ref doesn't work on Gateway account setting now. I open ticket for Recomm to fix this. I will change to ref after they fix that.

**Review by ashkarpetin — APPROVED** [review]

**Review by adamaltman — APPROVED** [review]

**Pull request merged**